README.md

```diff
@@ -317,21 +317,41 @@ Run `tns help` to view all available commands in the browser. Run `tns help <Com
 
 ## Create Project
 
-To create a new cross-platform project from the default template, run the following command.
+To create a new cross-platform project from the default JavaScript template, run the following command.
 
 ```Shell
 tns create MyApp
 ```
 
-To create a new cross-platform project from an existing NativeScript project, run the following command.
+To create a new cross-platform project from the default TypeScript or Angular template, use the `template` option followed by either `typescript`, or `angular`.
+
+```Shell
+tns create MyApp --template typescript
+tns create MyApp --template angular
+```
+
+Or you can simply use the shorthand `tsc` and `ng` options.
+
+```Shell
+tns create MyApp --tsc
+tns create MyApp --ng
+```
+With the `template` option you can also specify a local or a remote path to the template that you want to use to create your project.
+For example, if you want to use the nightly build of the default JavaScript template, run the following command.
```

Comment:
We don't have `@next` for the templates. Fetching the master branch from github is the only way to use the latest unofficial (or nightly in other words) template.